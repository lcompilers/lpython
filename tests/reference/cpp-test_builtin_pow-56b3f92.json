{
    "basename": "cpp-test_builtin_pow-56b3f92",
    "cmd": "lpython --no-color --show-cpp {infile}",
    "infile": "tests/../integration_tests/test_builtin_pow.py",
    "infile_hash": "73b7a50686598a8df8a84a6f36e637516e6f140b1d2adbc72f1c4cfe",
    "outfile": null,
    "outfile_hash": null,
    "stdout": null,
    "stdout_hash": null,
    "stderr": "cpp-test_builtin_pow-56b3f92.stderr",
    "stderr_hash": "e6d26ea9234e1f6b80909ee5e5b3506bfda19d62dfea9d4d2979b4af",
    "returncode": 3
}