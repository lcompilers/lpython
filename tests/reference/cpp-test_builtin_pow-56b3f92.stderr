style suggestion: Could have used '**' instead of 'pow'
  --> tests/../integration_tests/test_builtin_pow.py:11:16
   |
11 |     assert i32(pow(a, b)) == 32
   |                ^^^^^^^^^ '**' could be used instead
code generation error: Type number '16' not supported
   --> $DIR/src/bin/../runtime/lpython_builtin.py:106:1
    |
106 | T = TypeVar('T')
    | ^^^^^^^^^^^^^^^^ 
